File: IonicResearch.Data/Models/OutletPhoto.cs
using System.ComponentModel.DataAnnotations;

namespace IonicResearch.Data.Models
{
    public class OutletPhoto
    {
        public int Id { get; set; }
        [MaxLength(100)]
        public string Name { get; set; }
        [MaxLength(1000)]
        public string Path { get; set; }
        public int FormId { get; set; }

        public OutletForm Form { get; set; }
    }
}

File: IonicResearch.SPA/src/app/entities/outlet-photo.ts
import { Entity, PrimaryGeneratedColumn, Column, ManyToOne } from 'typeorm';
import { OutletForm } from './outlet-form';

@Entity('OutletPhoto')
export class OutletPhoto {
  @PrimaryGeneratedColumn()
  id: number;

  @Column({ length: 50 })
  name: string;

  @Column({ length: 1000 })
  path: string;

  @Column({ length: 1000 })
  filePath: string;

  @ManyToOne(type => OutletForm, form => form.photos, {
    cascade: true
  })
  form: OutletForm;
}
